dvc/fs/dvc.py

```diff
@@ -88,7 +88,7 @@ def __init__(
         target_options: Optional[Dict[str, Any]] = None,  # noqa: ARG002
         # pylint:disable-next=unused-argument
         target_protocol: Optional[str] = None,  # noqa: ARG002
-        **repo_kwargs: Any,
```

Comment:
fsspec `AbstractFileSystem.__init__()` needs to support arbitrary kwargs, and the fs is responsible for ignoring additional unexpected kwargs. We are not actually using `repo_kwargs` at all right now, if we need to forward new `Repo(...)` arguments into the `_make_repo` factory, we should explicitly add them in the future.

Reply:
minor correction, we do forward the repo+remote config dicts for repo imports now